src/endpoints/horde.js

```diff
@@ -364,12 +364,12 @@ router.post('/generate-image', jsonParser, async (request, response) => {
             return response.sendStatus(400);
         }
 
-        console.log('Horde image generation request:', generation);
+        console.debug('Horde image generation request:', generation);
```

Comment:
This should be an info. Here too this should be the "main log" showing that this endpoint was called and an image gen was requested.